chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/QuestionableActivityManager.java

```diff
@@ -206,6 +209,16 @@ public void checkListingQuestionableActivityOnEdit(CertifiedProductSearchDetails
                 createListingActivity(activity, origListing.getId(), activityDate, activityUser,
                         QuestionableActivityTriggerConcept.CRITERIA_B3_ADDED_TO_EXISTING_LISTING_WITH_ICS, activityReason);
             }
+            activity = listingQuestionableActivityProvider.checkRealWorldTestingPlanRemoved(origListing, newListing);
+            if (Objects.nonNull(activity)) {
```

Comment:
I don't like the inconsistency of having the other null checks for activity be `activity != null` but these two using the `Objects.nonNull`. It'd be one thing if they were in separate files, but here it just looks like we don't have a standard